src/components/Form.js

```diff
@@ -1,12 +1,24 @@
 import React from 'react';
 import Recaptcha from 'react-recaptcha';
+import Loader from 'react-loader-spinner';
 import TextInput from './TextInput';
 import RadioButton from './RadioButton';
 import Autocomplete from './Autocomplete';
 import Datepicker from './Datepicker';
 import formConfig from './formConfig.json';
 import { connect } from 'react-redux';
-import { handleFormSubmit } from '../redux/actions';
+import { Redirect } from 'react-router-dom';
+import { handleFormSubmit, formReset } from '../redux/actions';
+
+const LoadingState = () => (
+  <div className="pa7">
+    <Loader color="grey" width={200} heigth={80} type="RevolvingDot" />
+  </div>
+);
+
+const FieldWrapper = ({ children }) => (
```

Comment:
Created a component since a div with the same classes was being used to wrap all of the fields